lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_metadata.rb

```diff
@@ -42,6 +44,12 @@ def self.unpack(xMetadata, hResponseObj)
 
                   intMetadata[:resourceInfo] = ResourceInformation.unpack(xDataIdentification, hResponseObj)
 
+                  fileIdentifier = xMetadata.xpath(@@fileIdentifierXPath)[0]
+                  intMetadata[:fileIdentifier] = fileIdentifier.text unless fileIdentifier.nil?
+
+                  parentIdentifier = xMetadata.xpath(@@parentIdentifierXPath)[0]
+                  intMetadata[:parentIdentifier] = parentIdentifier.text unless parentIdentifier.nil?
```

Comment:
can you add a comment on each of these indicating if they're required/optional and the xsd element as evidence? comments like that don't exist entirely here because i made that decision after this module